src/test/kotlin/racingcar/domain/RacingGameTest.kt

```diff
@@ -0,0 +1,53 @@
+package racingcar.domain
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import racingcar.domain.numbergenerator.StubNumberGenerator
+import racingcar.view.InputView
+import racingcar.view.OutputView
+
+class RacingGameTest {
+    val inputView = InputView()
+    val outputView = OutputView()
+    val generator = StubNumberGenerator(mutableListOf(5, 3, 5))
+    val racingGame = RacingGame(inputView, outputView, generator)
+
+    @Test
+    fun `전진하는 경우`() {
```

Comment:
```suggestion
    fun `4이상이면 전진한다`() {
```
전진하는 경우가 과연 어떤 경우일까요?
구현부를 보지 않고 이름만 보고도 어떤 테스트인지 알 수 있도록 작성해보면 어떨까요? (이하 관련 내용 동일)